modules/dxp/apps/sharepoint-rest/sharepoint-rest-repository/src/main/java/com/liferay/sharepoint/rest/repository/internal/document/library/repository/external/SharepointRepositoryDefiner.java

```diff
@@ -83,6 +84,9 @@ public void registerCapabilities(
 				ProcessorCapability.ResourceGenerationStrategy.
 					ALWAYS_GENERATE));
 
+		capabilityRegistry.addExportedCapability(
+			ExtRepositoryCapability.class, new ExtRepositoryCapability());
+
```

Comment:
This shouldn't be needed. Check usages of `RepositoryUtil.isExternalRepository`.

Reply:
Fixed. (didn't know abut this method)

That's exactly what I needed.

Reply:
Fixed, I didn't remember that tool.

That's exactly what I need :)